pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "together"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
Bumping the minor version, because the behavior changes: if the user submits a JSONL file without a .jsonl extension, it will no longer work even if they do not check it